src/server/publicApi.js

```diff
@@ -113,6 +113,69 @@ const runPublicApi = (gameServer) => {
     ctx.body = model;
     });
 
+    //produce a nice textfile with the threats in
+    router.get('/download/text/:id', async (ctx) => {
+        //get some variables that might be useful
+        const gameName = ElevationOfPrivilege.name;
+        const gameID = ctx.params.id;
+        const res = await gameServer.db.get(`${gameName}:${gameID}`);
```

Comment:
What does `res` stand for?  result? Maybe there is a more descriptive name?

Reply:
I took this from the code that downloads the model so I'm not actually sure. I think maybe response or resource or sth. 

I'll come up with a better name